src/fetchers/stats-fetcher.js

```diff
@@ -44,14 +44,41 @@ const fetcher = (variables, token) => {
           followers {
             totalCount
           }
-          repositories(first: 100, ownerAffiliations: OWNER, orderBy: {direction: DESC, field: STARGAZERS}) {
+          repositories(ownerAffiliations: OWNER) {
```

Comment:
@MatteoPierro, I just noticed that in the old code we use 

```graphql
repositoriesContributedTo(first: 1, contributionTypes: [COMMIT, ISSUE, PULL_REQUEST, REPOSITORY]) {
            totalCount
          }
          pullRequests(first: 1) {
            totalCount
          }
```

Which can be simplified to:

```graphql
repositoriesContributedTo(contributionTypes: [COMMIT, ISSUE, PULL_REQUEST, REPOSITORY]) {
            totalCount
          }
          pullRequests {
            totalCount
          }
```

Could you maybe also apply these simplifications?

Reply:
Sure. Can I address this in a separated PR?

Reply:
For maintainability, it would be easier just to push a commit here.

Reply:
✅ Done